code/modules/antagonists/brother/brother.dm

```diff
@@ -9,6 +9,8 @@
 	suicide_cry = "FOR MY BROTHER!!"
 	antag_moodlet = /datum/mood_event/focused
 	antag_flags = parent_type::antag_flags | FLAG_ANTAG_CAP_TEAM // monkestation addition
+	var/datum/action/bb/comms/comms_action
+	var/datum/action/bb/gear/gear_action
 	VAR_PRIVATE
 		datum/team/brother_team/team
```

Comment:
Whoever wrote this should feel very bad about themselves. Just put it on 1 line zzz